<?xml version="1.0"?>
<launch>

	<!-- Set namespace -->
	<!--arg name="id_robot" default="robot"/-->

	<!-- Desired frequency -->
	<arg name="desired_freq" default="5.0" />
	<!-- Name of the example of a subscriber -->
	<arg name="example_subscriber_name" default="example" />
	<!-- Desired robot IP -->
	<arg name="punching_ip" default="192.168.17.195" />
	<!-- Desired robot port -->
	<arg name="punching_port" default = "502" />


	<!--group ns="$(arg id_robot)"-->

		<!-- start sirLifterDriver node -->
		<node name="punching_machine_driver" pkg="punching_machine_driver" type="punching_driver_node.py" output="screen" respawn="false">
			<param name="desired_freq" value="$(arg desired_freq)"/>
			<param name="example_subscriber_name" value="$(arg example_subscriber_name)"/>
			<param name="punching_ip" value="$(arg punching_ip)"/>
			<param name="punching_port" value="$(arg punching_port)"/>
			<!-- Load .yaml example -->
			<rosparam file="$(find punching_machine_driver)/config/modbusSignalsPEMU.yaml" command="load" />
		</node>

	<!--/group-->

</launch>
